src/backend/app.js

```diff
@@ -21,15 +22,72 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+app.get('/future-meals', async (req, res) => {
+
+  const [row] = await knex.raw(' SELECT * FROM `meals`WHERE `WHEN` > now()')
```

Comment:
Not sure if this missing space can give any issues
`meals`WHERE  -> `meals` WHERE